Assets/Scripts/MiniGames/Storms/RandomizerForStorms.cs

```diff
@@ -73,8 +75,15 @@ private void OnEnable()
 		{
 			Destroy(ship);
 		}
-		baseLighting = RenderSettings.ambientLight;
-		RenderSettings.ambientLight = stormLighting;
+
+		if (sunLight == null) {
+			sunLight = Globals.GameVars.skybox_sun.GetComponentInChildren<Light>().gameObject;
+		}
+
+		sunLight.SetActive(false);
+		Globals.GameVars.FPVCamera.gameObject.SetActive(false);
```

Comment:
i think we should move this into MiniGames.cs at some point so other minigames that switch to another camera can benefit from your fix. the board games the new team is working on will probably need to do this too. we can do that later though.